src/js/lib/youtube.js

```diff
@@ -13,7 +14,37 @@ export function pimpYouTubePlayer(videoId, node, height, width) {
                 events: {
                     'onReady': function(){
                       resolve(youTubePlayer);
-                    }
+                      // checkPlaybackTime(youTubePlayer);
+                    },
+                    'onStateChange': function(event){
+                      let chapTimer;
+                      if (event.data == YT.PlayerState.PLAYING) {
+
+                        var playerTotalTime = youTubePlayer.getDuration();
+                        chapTimer = setInterval(function() {
```

Comment:
should we test performance of this on mobile too; running this every second may be too much for mobile?